transformations/add_hashtag/test.json

```diff
@@ -0,0 +1,49 @@
+{
+  "type": "hashtag_generation",
+  "test_cases": [
+    {
+      "class": "HashtagGeneration",
+      "inputs": {
+        "sentence": "I love domino's pizza ."
+      },
+      "outputs": [{
+        "sentence": "I love domino's pizza .#Love#Pizza"
+      }, {
+        "sentence": "I love domino's pizza .#Pizza#Domino'SPizza"
+      }]
+    },
+    {
+      "class": "HashtagGeneration",
+      "inputs": {
+        "sentence": "Many people like T20 cricket these days ."
+      },
+      "outputs": [{
+        "sentence": "Many people like T20 cricket these days .#People#Cricket#T20"
```

Comment:
Would it be a good idea to tag even hypernyms or some words which are not in the original sentence?